apps/backoffice-v2/src/common/utils/export-to-csv/export-to-csv.unit.test.ts

```
@@ -0,0 +1,195 @@
+      expect(actual).toMatch(/John, Doe/);
+      expect(actual).toMatch(/He said:.*Hello/);
+      expect(actual).toContain('Jane Doe');
+      expect(actual).toContain('Normal text');
+    });
+  });
+
+  describe('exportToCSV', () => {
+    // Setup for DOM mocking
+    let originalCreateObjectURL: typeof URL.createObjectURL;
+    let originalRevokeObjectURL: typeof URL.revokeObjectURL;
+    let mockLink: { href: string; setAttribute: Function; click: Function };
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)